R/tm_g_scatterplotmatrix.R

```diff
@@ -461,26 +463,29 @@ srv_g_scatterplotmatrix <- function(id, data, reporter, filter_panel_api, variab
 }
 
 #' Get stats for x-y pairs in scatterplot matrix
-#' @description uses stats::cor.test per default for all numerical input variables and converts results
-#'  to character vector. Could be extended if different stats for different variable
-#'  types are needed. Meant to be called from \code{lattice::panel.text}.
-#' @param x \code{numeric}
-#' @param y \code{numeric}
-#' @param .f \code{function}, function that accepts x and y as formula input \code{~ x + y}.
-#' Default \code{stats::cor.test}
-#' @param .f_args \code{list} of arguments to be passed to \code{.f}
-#' @param round_stat \code{integer}
-#' @param round_pval \code{integer}
-#' @details presently we need to use a formula input for \code{stats::cor.test} because
-#' \code{na.fail} only gets evaluated when a formula is passed (see below).
-#' \preformatted{
+#'
+#' Uses `stats::cor.test` per default for all numerical input variables and converts results
+#' to character vector.
+#' Could be extended if different stats for different variable types are needed.
+#' Meant to be called from `lattice::panel.text`.
+#'
+#' Presently we need to use a formula input for `stats::cor.test` because
+#' `na.fail` only gets evaluated when a formula is passed (see below).
+#' ```
 #' x = c(1,3,5,7,NA)
 #' y = c(3,6,7,8,1)
 #' stats::cor.test(x, y, na.action = "na.fail")
 #' stats::cor.test(~ x + y,  na.action = "na.fail")
-#' }
-#' @return \code{character} with stats. For \code{stats::cor.test} correlation coefficient and p-value.
-#' @export
+#' ```
+#' @param x (`numeric`)
+#' @param y (`numeric`)
```

Comment:
Should there be a description here?

(suggestion copies it from `stats::corr` and adds the data type)

```suggestion
#' @param x,y (`numeric`) vectors of data values. `x` and `y` must have the same length.
```